corehq/apps/styleguide/example_forms.py

```diff
@@ -24,7 +23,7 @@ class BasicCrispyForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         super(BasicCrispyForm, self).__init__(*args, **kwargs)
-        self.helper = FormHelper()
+        self.helper = hqcrispy.HQFormHelper()
 
         self.helper.form_method = 'POST'
         self.helper.form_class = 'form-horizontal'
```

Comment:
I'm not sure about form_method, but I think you can take out the labels,fields and form class? ( applicable to all instances you changed)

Reply:
a74811a3737